src/commonMain/kotlin/exchange.dydx.abacus/state/manager/utils/Payloads.kt

```diff
@@ -20,6 +21,7 @@ data class PlaceOrderRecord(
     val clientId: Int,
     val timestampInMilliseconds: Double,
     val fromSlTpDialog: Boolean,
+    var lastOrderStatus: OrderStatus?,
```

Comment:
is this.. legal? :D

Reply:
nit I wonder if we can just do `val lastOrderStatus: OrderStatus? = null`

Reply:
the main issue is i can't reassign lastOrderStatus if it's `val` instead of `var`, but i can add the =null there